application/src/main/java/com/softserve/lv460/application/controller/HomeController.java

```diff
@@ -79,11 +71,11 @@
 
   @ApiOperation(value = "Return list of home by user")
   @ApiResponses(value = {
-    @ApiResponse(code = 200, message = HttpStatuses.OK, response = HomeResponseDTO.class)
+        @ApiResponse(code = 200, message = HttpStatuses.OK, response = HomeResponseDTO.class)
   })
-  @GetMapping("/user/{user_id}")
-  public ResponseEntity<List<HomeResponseDTO>> findAllByUserId(@PathVariable("user_id") Long userId) {
-    return ResponseEntity.status(HttpStatus.OK).body(homeService.findAllByUser(userId).stream().map(responseMapper::toDto)
-      .collect(Collectors.toList()));
+  @GetMapping
+  public ResponseEntity<List<HomeResponseDTO>> findAllByUserId(@CurrentUser UserPrincipal user) {
```

Comment:
It's better to rename method to "findAllByUser" because of method accept user, not id